content/c/concepts/memory-management/terms/realloc/realloc.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'realloc()'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Description: ''
+Tags:
+  - 'Memory'
+  - 'Memory Management'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+The **`realloc()`** function is used to reallocate a block of memory. If the memory that was previously allocated using the `malloc()` or `calloc()` function is not sufficient, so it can be reallocated using the `realloc()` function.
```

Comment:
```suggestion
The **`realloc()`** function is used to reallocate a block of memory. If the memory that was previously allocated using the `malloc()` or `calloc()` function is not sufficient, it can be reallocated using the `realloc()` function.
```